gbdxtools/simpleworkflows.py

```diff
@@ -255,6 +253,21 @@ def list_workflow_outputs(self):
 
         return workflow_outputs
 
+    def as_json(self):
```

Comment:
The workflow is output into the log for diagnostic purposes.  I've removed this on merge, however, and will use that instead.